sql/src/test/java/org/opensearch/sql/sql/antlr/SQLSyntaxParserTest.java

```diff
@@ -175,6 +175,37 @@ private static Stream<Arguments> nowLikeFunctionsData() {
     );
   }
 
+  private static Stream<Arguments> getPartForExtractFunction() {
+    return Stream.of(
+        Arguments.of("MICROSECOND"),
```

Comment:
Probably need a UT or IT with incorrect `part` argument:
```sql
select extract(cat from date '1984-04-04')
```

Reply:
Added in https://github.com/opensearch-project/sql/pull/1421/commits/a38212cbef93f40de3e26ff6628fcfdd9b9660c0